helix-view/src/input.rs

```diff
@@ -13,6 +13,32 @@ pub struct KeyEvent {
     pub modifiers: KeyModifiers,
 }
 
+pub(crate) mod keys {
+    pub(crate) const BACKSPACE: &str = "backspace";
+    pub(crate) const ENTER: &str = "ret";
+    pub(crate) const LEFT: &str = "left";
+    pub(crate) const RIGHT: &str = "right";
+    pub(crate) const UP: &str = "up";
+    pub(crate) const DOWN: &str = "down";
+    pub(crate) const HOME: &str = "home";
+    pub(crate) const END: &str = "end";
+    pub(crate) const PAGEUP: &str = "pageup";
+    pub(crate) const PAGEDOWN: &str = "pagedown";
+    pub(crate) const TAB: &str = "tab";
+    pub(crate) const BACKTAB: &str = "backtab";
+    pub(crate) const DELETE: &str = "del";
+    pub(crate) const INSERT: &str = "ins";
+    pub(crate) const NULL: &str = "null";
+    pub(crate) const ESC: &str = "esc";
+    pub(crate) const SPACE: &str = "space";
+    pub(crate) const LESS_THAN: &str = "lt";
+    pub(crate) const GREATER_THAN: &str = "gt";
+    pub(crate) const PLUS: &str = "plus";
```

Comment:
Shouldn't all these just be symbols? `+`, `-`, etc

Reply:
No, this follows the same as how we do parsing, we would not want to have a separated key identification which will confuse the users. Imagine we use `+` for output and `plus` for input.

Reply:
Personally I would prefer symbols in both. Was there an issue with symbols?

Reply:
Same here. I missed this change in the original PR but is that because of emacs-like key names?

Reply:
No, the name I think is based on kakoune implementation. I do think using symbol will be easier so users don't have to do a lookup.

Reply:
I changed it to use how we parse stuff. Maybe if we want to change it need to change parsing.